diagnostic.md

```diff
@@ -16,13 +16,14 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/Gabrakadabra/wdi/diagnostics/fundamentals-diagnostic
+~/wdi/diagnostics/fundamentals-diagnostic
```

Comment:
The current working directory for this example is /Users/grace/wdi/try.  After running pwd, we created the "try" directory and then entered it with "cd try".